pkgs/by-name/ui/uiua/package.nix

```diff
@@ -37,6 +41,13 @@ rustPlatform.buildRustPackage rec {
 
   buildFeatures = lib.optional audioSupport "audio";
 
+  passthru.tests.run = runCommand "uiua-test-run" {nativeBuildInputs = [uiua];} ''
+    uiua init;
+    a=$(uiua run main.ua);
+    [[ $a == '"Hello, World!"' ]];
```

Comment:
This won't show the diff like the hello one does

Reply:
ohh, I see, I had tunnelvision just making a test, doing it with the diff is much nicer!